WooCommerce/WooCommerceTests/ViewRelated/Orders/OrderListViewModelTests.swift

```diff
@@ -252,40 +252,17 @@ final class OrderListViewModelTests: XCTestCase {
                 break
             }
         }
-        viewModel.activate()
 
         // When
-        viewModel.updateBannerVisibility()
-
-        // Then
-        waitUntil {
-            viewModel.topBanner == .none
-        }
-    }
-
-    func test_when_having_no_error_and_orders_banner_should_be_shown_then_shows_orders_banner() {
-        // Given
-        let viewModel = OrderListViewModel(siteID: siteID, stores: stores, filters: nil)
-        stores.whenReceivingAction(ofType: AppSettingsAction.self) { action in
-            switch action {
-            case let .loadFeedbackVisibility(_, onCompletion):
-                onCompletion(.success(true))
-            default:
-                break
-            }
-        }
         viewModel.activate()
 
-        // When
-        viewModel.updateBannerVisibility()
-
         // Then
         waitUntil {
```

Comment:
We don't need to call `waitUntil` because there is no async work to wait for here.